jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -65,11 +72,69 @@ def shell_context():
     app.shell_context_processor(shell_context)
 
 
-if __name__ == '__main__':
-    application = create_app()
+def initiate_dissolution_process(app: Flask):  # pylint: disable=redefined-outer-name
+    """Initiate dissolution process for new businesses where AR has not been filed for 2 yrs and 2 months."""
```

Comment:
I think I use "Initiate dissolution process for new businesses where AR has not been filed for 2 yrs and 2 months." as the title of stage 1 in my tech design.  That is a little inaccurate are other possible selection criteria.  

Should probably just update this to something like "Initiate dissolution process for new businesses that meet dissolution criteria"